code/src/index.css

```diff
@@ -11,3 +14,205 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* - - - MovieList.js - - - */
+
+.movie {
+  display: grid;
+  align-items: end;
+  position: relative;
+}
+
+.poster-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+.poster-image {
+  width: 100%;
+}
+
+.poster-image, .poster-text-container {
+  transition: opacity 300ms ease-in-out;
+}
+
+.poster-text-container {
+  position: absolute;
+  margin: calc(15px + 1vw);
```

Comment:
Interesting, what does calc do here?

Reply:
It sets the value to be 15 px plus 1% of the viewport width, resulting in a margin that adjusts slightly depending on window size but typically ends up being around 25 px